src/containers/HeaderRedesignWrapper/HeaderRedesignWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Header from '@components/HeaderRedesign/Header'
+import { useDispatch, useSelector } from 'react-redux'
+import { useLocation } from 'react-router-dom'
+import { address, status } from '@selectors/solanaWallet'
+import { actions } from '@reducers/solanaConnection'
+import { SolanaNetworks } from '@consts/static'
+import { Status, actions as walletActions } from '@reducers/solanaWallet'
+import { WalletType } from '@web3/wallet'
+
+export const HeaderRedesignWrapper: React.FC = () => {
+  const dispatch = useDispatch()
+  const walletAddress = useSelector(address)
+  const walletStatus = useSelector(status)
+  const location = useLocation()
+
+  const stringToWalletType = (chosen: string) => {
+    if (chosen === 'phantom') {
```

Comment:
Use Enum as a parameter instead of additional one.

Reply:
done